lib/wasabi/parser.rb

```diff
@@ -117,7 +117,7 @@ def parse_port_type_operations
     end
 
     def parse_operations_parameters
-      root_elements = document.xpath("wsdl:definitions/wsdl:types/*[local-name()='schema']/*[local-name()='element']", 'wsdl' => WSDL).each do |element|
+       document.xpath("wsdl:definitions/wsdl:types/*[local-name()='schema']/*[local-name()='element']", 'wsdl' => WSDL).each do |element|
```

Comment:
```suggestion
      document.xpath("wsdl:definitions/wsdl:types/*[local-name()='schema']/*[local-name()='element']", 'wsdl' => WSDL).each do |element|
```

This whitespace seemed to be wonky.